tests/test_yamls/use_intermediate_bucket.yaml

```diff
@@ -0,0 +1,5 @@
+jobs:
```

Comment:
Looks like this is a SkyPilot config YAML. Should we rename this file to `use_intermediate_bucket_config.yaml` to distinguish from other Task YAMLs?